### PR TITLE
feat(multiedit): Multiedit service

### PR DESCRIPTION
**Pull request by nidzix**

### CODE REVIEW

**Comment by nidzix** [comment]
I'm not happy with how I started implementation. Any feedback? @petrjasek @amagdas

File: app/scripts/superdesk-authoring/multiedit/multiedit.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+					}
+					if (scope.selected(_id)) {
+						scope.queue = _.without(scope.queue, _id);
+					} else {
+						scope.queue.push(_id);
+					}
+				};
+
+				scope.selected = function(_id) {
+					return _.indexOf(scope.queue, _id) === -1 ? false : true;
```
Comment (amagdas) [inline app/scripts/superdesk-authoring/multiedit/multiedit.js]: `_.indexOf(scope.queue, _id) !== -1`

**Comment by amagdas** [comment]
Looks decent to me, still I would like to see this whole feature in a separate feature branch as we used for packages.

File: app/scripts/superdesk-authoring/multiedit/multiedit.js (hunk 1 above)
Comment (nidzix) [inline app/scripts/superdesk-authoring/multiedit/multiedit.js]: :disappointed:

**Comment by nidzix** [comment]
Not sure if we should allow stack of multiedit screens, or go with single multiedit screen....

**Comment by amagdas** [comment]
We can start with only one and accommodate for more if really needed...

**Comment by petrjasek** [comment]
so are we going to have multiple multiedit screens?

**Comment by nidzix** [comment]
This implementation assumes so, but it complicates things a lot. Even though need for multiple multedit screens will probably be minor.

**Comment by petrjasek** [comment]
I think we should leave it out then. this whole thing is a prototype so making it less complicated now is better

**Comment by petrjasek** [comment]
so what about closing this PR and creating another on a new branch?

**Comment by amagdas** [comment]
Yup :+1:

**Comment by nidzix** [comment]
:ok:

**Pull request closed without merging**